microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -611,4 +611,13 @@ Issued: 2024-09-12
 
 {product-title} release 4.14.36 is now available. The list of enhancements and bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:6410[RHBA-2024:6410] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:6406[RHSA-2024:6406] advisory.
 
+For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
+
+[id="microshift-4-14-37-dp_{context}"]
+=== RHBA-2024:6690 - {microshift-short} 4.14.37 bug fix and security update advisory
+
+Issued: 2024-09-19
```

Comment:
I know this is consistent with the MS template for RNs but I wanted to point out that OCP core has switched to the following [guidelines](https://docs.openshift.com/container-platform/4.16/release_notes/ocp-4-16-release-notes.html#ocp-4-16-11_release-notes:~:text=Issued%3A-,11%20September%202024,-OpenShift%20Container%20Platform) for dates (ISG).

Reply:
Done. Thank you for pointing out.